### PR TITLE
[testcase] Move Orderby cases to tck

### PR DESCRIPTION
**Pull request by jude-zhu**

### CODE REVIEW

File: tests/tck/features/go/Orderby.feature [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      GO FROM "Boris Diaw" OVER serve YIELD $^.player.name as name, serve.start_year as start, $$.team.name as team | ORDER BY $-.abc
+      """
+    Then a SemanticError should be raised at runtime:
+
+  Scenario: Sorting results using single column
+    When executing query:
+      """
+      GO FROM "Boris Diaw" OVER serve YIELD $^.player.name as name, serve.start_year as start, $$.team.name as team | ORDER BY $-.team
+      """
+    Then the result should be, in any order, with relax comparison:
```
Comment (whitewum) [inline tests/tck/features/go/Orderby.feature:54]: why relax comparison?

Comment (yixinglu) [reply tests/tck/features/go/Orderby.feature:54]: should be in order

File: tests/tck/features/go/Orderby.feature [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      | "Boris Diaw" | 2008  | "Hornets" |
+      | "Boris Diaw" | 2003  | "Hawks"   |
+
+  Scenario: Sorting using multi columns
+    When executing query:
+      """
+      GO FROM "Boris Diaw","LaMarcus Aldridge" OVER serve WHERE serve.start_year >= 2012 YIELD $$.team.name as team, $^.player.name as player,
+      $^.player.age as age, serve.start_year as start | ORDER BY $-.team, $-.age
+      """
+    Then the result should be, in any order, with relax comparison:
```
Comment (yixinglu) [inline tests/tck/features/go/Orderby.feature:90]: in order

File: tests/tck/features/go/Orderby.feature [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+      GO FROM "Boris Diaw" OVER serve YIELD $^.player.name as name, serve.start_year as start, $$.team.name as team | ORDER BY $-.abc
+      """
+    Then a SemanticError should be raised at runtime:
+
+  Scenario: Sorting results using single column
+    When executing query:
+      """
+      GO FROM "Boris Diaw" OVER serve YIELD $^.player.name as name, serve.start_year as start, $$.team.name as team | ORDER BY $-.team
+      """
+    Then the result should be, in order, with relax comparison:
```
Comment (yixinglu) [inline tests/tck/features/go/Orderby.feature:54]: In this feature file,  the `the result should be, in order` step is more reasonable.

**Review by yixinglu — APPROVED** [review]

**Review by laura-ding — APPROVED** [review]

**Pull request merged**